chicago/permit_cleaning.py

```diff
@@ -748,11 +747,11 @@ def save_xlsx_files(df, max_rows, file_base_name):
     ]
 
     file_name_combined = os.path.join(
-        folder_for_files_review, file_base_name + "review.xlsx"
+        folder_for_files_review, file_base_name + "needing_review.xlsx"
     )
 
     # Copy template workbook
-    template_file = "chicago/templates/permits_needing_review.xlsx"
+    template_file = os.path.join("templates", "permits_needing_review.xlsx")
```

Comment:
`os.path.join()` is the Pythonic interface for constructing file paths. It's useful because different file systems use different directory separators, e.g. Unix uses forward slash `/` but Windows uses backslash `\`.